docs/messaging/gleanplumb.mdx

```diff
@@ -146,6 +152,40 @@ If `title` is missing, then the title is not displayed.
 
 If `button-label` is missing, then the button is not displayed, but the whole message is tappable.
 
+### Surfaces
+
+The `surface` of the message is the message surface on which the message will be displayed.
+
+These are app specific: the application provides the surface to draw the message on.
+
+<Tabs
+  defaultValue="fenix"
+  values={[
+    { label: "Firefox for Android", value: "fenix" },
+    { label: "Firefox for iOS", value: "fxios" },
+  ]
+}>
+<TabItem value="fenix">
+
+Surface | Description | Versions |
+----------|------|-------------|
+`homescreen`    | On the new tab/homescreen                                             | |
+`notification`  | A system notification. Permission is required for Android SDK > 13    | v111 |
+
+</TabItem>
+<TabItem value="fxios">
+
+Surface | Description | Versions |
+----------|------|-------------|
+`homescreen-banner`| On the new tab/homescreen      | |
+
+While there is only one surface, the surface attribute can be omitted.
+
+It is possible this table is out of date. The definitive source of truth for this in [the code][fxios-attributes-eng] itself.
```

Comment:
The named link here is a missing link, do we have an updated link we can use?

Also, I think we should have this line under the Android tab as well.